client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/constants.ts

```diff
@@ -68,6 +68,8 @@ export const ORDERED_PATTERN_CATEGORIES = [
 	'contact',
 ];
 
+export const INITIAL_PAGE = 'about';
```

Comment:
[nit] I think it could be an array to include any preselected pages 🙂

Reply:
Sure!